src/main/java/com/google/ehub/data/EntertainmentItem.java

```diff
@@ -9,6 +11,20 @@
   private final String description;
   private final String imageURL;
 
+  /**
+   * Creates EntertainmentItem with a new unique ID.
+   *
+   * @param title the title used to construct the EntertainmentItem
+   * @param description the description used to construct the EntertainmentItem
+   * @param imageURL the image URL used to contruct the EntertainmentItem
+   * @return new EntertainmentItem with unique ID
+   */
+  public static EntertainmentItem unassignedItem(
+      String title, String description, String imageURL) {
+    return new EntertainmentItem(
+        UUID.randomUUID().getMostSignificantBits() & Long.MAX_VALUE, title, description, imageURL);
```

Comment:
This is a little confusing because its still assigning a unique id to the item.

If might be better to make uniqueId optional so users know that Optional.empty() means the id was never assigned.
Another option is to only create an `EntertainmentItem` object when the data is queried from the datastore. This guarantees that the uniqueId is always populated.

Reply:
Yeah I removed the need to create a unique Id in the static method, now Datastore is used to deal with that.